packages/jsii-rosetta/test/commands/transliterate.test.ts

```diff
@@ -139,9 +139,6 @@ export class ClassName implements IInterface {
           "markdown": "# README
 
       \`\`\`csharp
-      // This example was automatically transliterated.
```

Comment:
Why did these disclaimers disappear all of a sudden? Are we not compiling (in which case we should), or did we not switch on `prefixDisclaimer` (which we also should)